src/main/java/com/assignment/application/Constants/StringConstants.java

```diff
@@ -0,0 +1,18 @@
+package com.assignment.application.Constants;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class StringConstants {
```

Comment:
Make this class a singleton with private constructor. No need to make a bean as well. :)